docs/modules/components/pages/inputs/timeplus.adoc

```diff
@@ -0,0 +1,167 @@
+= timeplus
+:type: input
+:status: experimental
+:categories: ["Services"]
+
+
+
+////
+     THIS FILE IS AUTOGENERATED!
+
+     To make changes, edit the corresponding source file under:
+
+     https://github.com/redpanda-data/connect/tree/main/internal/impl/<provider>.
+
+     And:
+
+     https://github.com/redpanda-data/connect/tree/main/cmd/tools/docs_gen/templates/plugin.adoc.tmpl
+////
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Executes a query on Timeplus Enterprise and creates a message from each row received
+
+```yml
+# Config fields, showing default values
+input:
+  label: ""
+  timeplus:
+    query: select * from iot # No default (required)
+    url: tcp://localhost:8463
+    workspace: "" # No default (optional)
+    apikey: "" # No default (optional)
+    username: "" # No default (optional)
+    password: "" # No default (optional)
+```
+
+This input can execute a query on Timeplus Enterprise Cloud, Timeplus Enterprise (self-hosted) or Timeplusd. A structured message will be created
```

Comment:
Maybe changing the 3rd option from `Timeplusd` to `Timeplus Proton`. It's a bit confusing what's the difference in this context for 2nd and 3rd option.